src/transaction.js

```diff
@@ -46,447 +31,525 @@ const BLANK_OUTPUT = {
   valueBuffer: VALUE_UINT64_MAX
 }
 
-Transaction.fromBuffer = function (buffer, __noStrict) {
-  let offset = 0
-  function readSlice (n) {
-    offset += n
-    return buffer.slice(offset - n, offset)
+class Transaction {
+  constructor () {
+    this.version = 1
+    this.locktime = 0
+    this.ins = []
+    this.outs = []
   }
 
-  function readUInt32 () {
-    const i = buffer.readUInt32LE(offset)
-    offset += 4
-    return i
+  static get DEFAULT_SEQUENCE () {
+    return 0xffffffff
```

Comment:
is this _the_ way to do constants?

Reply:
This is the best way I found. Open to other suggestions. (I'm guessing you want one constant per line?)